src/main/java/core/basesyntax/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The class should not start with an empty line. Please remove the redundant empty line at the beginning of the class definition.